### PR TITLE
Add 'Who is your pension provider?' field to appointments/new

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: app/helpers/appointment_helper.rb [hunk 1]
```diff
@@ -17,13 +17,52 @@ module AppointmentHelper
...
+      WHO_IS_YOUR_PENSION_PROVIDER_OPTIONS,
+      appointment
+    )
+  end
+
+  private
+
+  def select_options(field, options, object)
+    options |=
+      [object.send(field)]
```
Comment (benlovell) [inline app/helpers/appointment_helper.rb]: We should favour `public_send` here and below.

Comment (AndrewVos) [inline app/helpers/appointment_helper.rb]: Done, and also reordered the method parameters to be `(object, field, options)`. Seemed more logical.

Comment (dwhenry) [inline app/helpers/appointment_helper.rb]: Do we need to use send. 

Would it be easier to just pass the value in given we only use the field to extract the value from the object (which we also pass in)

Comment (AndrewVos) [inline app/helpers/appointment_helper.rb]: Yeah that sounds good to me... Done. @dwhenry

**Pull request merged**